publicHTML/public/index.html

```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title></title>
+</head>
+<body>
+  <FORM action="/send" method="GET" name="getRequest">
```

Comment:
don't all caps form. HTML tags should be lower cased. in vim, simply type `form` then ctrl+y then ,